tests/integration/local/invoke/test_integrations_cli_cdk.py

```diff
@@ -0,0 +1,200 @@
+import json
+import shutil
+import os
+import copy
+import tempfile
+from distutils.dir_util import copy_tree
+from unittest import skipIf
+
+from parameterized import parameterized, parameterized_class
+from subprocess import Popen, PIPE, TimeoutExpired
+from timeit import default_timer as timer
+import pytest
+import docker
+
+from tests.integration.local.invoke.layer_utils import LayerUtils
+from .invoke_integ_base import InvokeIntegBase, CDKInvokeIntegPythonBase
+from tests.testing_utils import IS_WINDOWS, RUNNING_ON_CI, RUNNING_TEST_FOR_MASTER_ON_CI, RUN_BY_CANARY
+
+# Layers tests require credentials and Appveyor will only add credentials to the env if the PR is from the same repo.
+# This is to restrict layers tests to run outside of Appveyor, when the branch is not master and tests are not run by Canary.
+SKIP_LAYERS_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
+
+from pathlib import Path
+
+TIMEOUT = 300
+
+
+class TestCdkPythonHelloWorldIntegration(CDKInvokeIntegPythonBase):
+    @pytest.mark.flaky(reruns=0)
+    def test_invoke_returncode_is_zero(self):
+        test_data_path = self.test_data_path.joinpath("cdk", "python", "aws-lambda-function")
+        copy_tree(test_data_path, self.working_dir)
```

Comment:
Do we need to run `self.cdk_python_env.install_dependencies()` to ensure that cdk dependencies are install?

Reply:
Thanks for metioning it! Added it in the class setup since the requirements are the same in the test cases.